### ISSUE TITLE
[Bug] Blank screen upon program launch

### ISSUE THREAD

**Issue by aquamarine-axo** [Vencord/Vesktop#664]
### Discord Account

aquamarine.gay

### Operating System

Arch Linux x86_64 (all packages are fully updated)

### Linux Only ~ Desktop Environment

Sway on Wayland

### What happens when the bug or crash occurs?

When launching the program, either through the official AppImage or the AUR package (vesktop_electron), the program launches with a completely gray window. However, it's not a GPU issue because you can still open the Dev Tools console (with the exception of saving the console logs, which segfaults in the AUR version but not the AppImage version).

### What is the expected behaviour?

I expected the regular Discord interface to appear shortly after the application launches.

### How do you recreate this bug or crash?

1. Install any Vesktop AUR package, or launch the official AppImage (Flatpak untested)
2. Observe gray screen in place of normal Discord UI

### Debug Logs

NOTE: Any notes I've left (such as me doing something, causing a log to appear) are commented out with a hashtag.

**Console logs (in terminal)**
```
[3308:0617/152851.738416:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
[3308:0617/152853.419238:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 2 times!
[3308:0617/152855.629999:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 3 times!
# I opened the DevTools flyout here
[3266:0617/152856.128220:ERROR:CONSOLE(1)] "Request Autofill.enable failed. {"code":-32601,"message":"'Autofill.enable' wasn't found"}", source: devtools://devtools/bundled/core/protocol_client/protocol_client.js (1)
```

**Console logs (in DevTools)**
```
Logger.ts:40  Vencord   PluginManager  Starting plugins (stage Init)
VCDRenderer:1 read if cute :3
app:44 [FAST CONNECT] wss://gateway.discord.gg/?encoding=json&v=9&compress=zlib-stream, encoding: json, version: 9
Logger.ts:40  Vencord   WebpackInterceptor  Patching webpackChunkdiscord_app.push
app:44 [FAST CONNECT] connected in 196ms
Logger.ts:40  Vencord   WebpackInterceptor  Patch by PermissionFreeWill had no effect (Module id is 156699): /(?<=(?:isDefaultChannelThresholdMetAfterDelete|checkDefaultChannelThresholdMetAfterChannelPermissionDeny):function\(\)\{)return [A-Za-z_$][\w$]*(?=\})/g
_log @ Logger.ts:40
Logger.ts:40  Vencord   WebpackInterceptor  Patch by Vesktop had no effect (Module id is 292959): /\.isPlatformEmbedded(?=\?[A-Za-z_$][\w$]*\.DesktopNotificationTypes\.ALL)/g
_log @ Logger.ts:40
Logger.ts:40  Vencord   WebpackInterceptor  Patch by Vesktop had no effect (Module id is 649318): /"x-google-max-bitrate=".concat\([A-Za-z_$][\w$]*\)/
_log @ Logger.ts:40
Logger.ts:40  Vencord   WebpackInterceptor  Patch by BadgeAPI had no effect (Module id is 741308): /(?<=([A-Za-z_$][\w$]*)=\(0,[A-Za-z_$][\w$]*\.default\)\([A-Za-z_$][\w$]*\);)return 0===[A-Za-z_$][\w$]*.length\?/
_log @ Logger.ts:40
Logger.ts:40  Vencord   WebpackInterceptor  Patch by RoleColorEverywhere had no effect (Module id is 845537): /let\{id:([A-Za-z_$][\w$]*),guildId:([A-Za-z_$][\w$]*)[^}]*\}.*?\.default,{(?=children)/
_log @ Logger.ts:40
Logger.ts:40  Vencord   WebpackInterceptor  Patch by WebContextMenus had no effect (Module id is 959856): /supports\([A-Za-z_$][\w$]*\)\{switch\([A-Za-z_$][\w$]*\)\{case ([A-Za-z_$][\w$]*).Features/
_log @ Logger.ts:40
Logger.ts:40  Vencord   WebpackInterceptor  Found Webpack module factory web.dede0f630537b26553d1.js
Logger.ts:40  Vencord   WebpackInterceptor  Found main WebpackRequire.onChunksLoaded
Logger.ts:40  Vencord   WebpackInterceptor  Patched initialize app callback invoked, initializing our internal references to WebpackRequire and running beforeInitListeners
25624.57396d670824cfdaa9f8.js:1 [ProcessUtilsElectron] setMemoryInformation not available.
25624.57396d670824cfdaa9f8.js:1 [default] [BUILD INFO] Release Channel: canary, Build Number: 302575, Version Hash: b5e8282b6d80e92c757b9f9495f8fd194eacf58b
25624.57396d670824cfdaa9f8.js:1 AppCrashedFatalReport: getLastCrash not supported.
25624.57396d670824cfdaa9f8.js:1 Window state not initialized undefined
I @ 25624.57396d670824cfdaa9f8.js:1
Logger.ts:40  Vencord   WebpackInterceptor  Found Webpack module factory sentry.8713026a9351b2082068.js
Logger.ts:40  Vencord   PluginManager  Starting plugins (stage DOMContentLoaded)
25624.57396d670824cfdaa9f8.js:1 Uncaught (in promise) Error: Cannot find module './notosans-400-normalitalic.woff2'
    at 25624.57396d670824cfdaa9f8.js:1:10623
    at async Promise.all (index 3)
Logger.ts:40  Vencord   WebpackInterceptor  Patch by RoleColorEverywhere had no effect (Module id is 785388): /\.openUserContextMenu\)\(([A-Za-z_$][\w$]*),([A-Za-z_$][\w$]*),[A-Za-z_$][\w$]*\).{0,250}tag:"strong"/
_log @ Logger.ts:40
Logger.ts:40  Vencord   WebpackInterceptor  Patch by FakeNitro had no effect (Module id is 217588): /let{renderableSticker:([A-Za-z_$][\w$]*).{0,250}isGuildSticker.+?channel:[A-Za-z_$][\w$]*,/
_log @ Logger.ts:40
Logger.ts:40  Vencord   WebpackInterceptor  Patch by Vesktop had no effect (Module id is 897607): /([A-Za-z_$][\w$]*)\.default\.getState\(\).neverShowModal/
_log @ Logger.ts:40
67096.68638f0c217ebd5f8c80.js:12 [DEPRECATED] Please use `subscribeWithSelector` middleware
a @ 67096.68638f0c217ebd5f8c80.js:12
Logger.ts:40  Vencord   Webpack  Error: webpack.findStore found no module
    at xn (webpack.ts:92:17)
    at Rp (webpack.ts:339:9)
    at webpack.ts:349:28
    at [vencord.lazy.get] (lazy.ts:98:47)
    at Object.get (lazy.ts:122:39)
    at isActivityNotIgnored (index.tsx:273:46)
    at Array.filter (<anonymous>)
    at D (WebpackModule293273:2:1613)
    at START_SESSION (WebpackModule293273:2:2364)
    at LocalActivityStore_START_SESSION (25624.57396d670824cfdaa9f8.js:1:7110884) Filter: Array(1)
_log @ Logger.ts:40
lazy.ts:124 Uncaught (in promise) TypeError: Reflect.get called on non-object
    at Reflect.get (<anonymous>)
    at Object.get (lazy.ts:124:32)
    at isActivityNotIgnored (index.tsx:273:46)
    at Array.filter (<anonymous>)
    at D (WebpackModule293273:2:1613)
    at START_SESSION (WebpackModule293273:2:2364)
    at LocalActivityStore_START_SESSION (25624.57396d670824cfdaa9f8.js:1:7110884)
    at 25624.57396d670824cfdaa9f8.js:1:7109381
    at 25624.57396d670824cfdaa9f8.js:1:7115901
    at T._dispatch (25624.57396d670824cfdaa9f8.js:1:7109373)
```

### Request Agreement

- [X] I have searched the existing issues and found no similar issue
- [X] I am using the latest Vesktop and Vencord versions
- [X] This issue occurs on an official release (not just the AUR or Nix packages)

**Comment by Vendicated** [comment]
canary is broken. use stable for now

you can change via devtools (ctrl shift i):
```js
Vesktop.Settings.store.discordBranch = "stable"
```
then fully restart vesktop

**Comment by aquamarine-axo** [comment]
Above comment worked. Thanks

**Comment by Vendicated** [comment]
this should stay open until the issue is fully resolved in vencord, so other users also see it

**Comment by amykatname** [comment]
appreciate it!!

**Comment by Vendicated** [comment]
now fully fixed, see #669
